generate-community/src/bin/generate.rs

```diff
@@ -1,65 +1,118 @@
+            .unwrap();
     let role_map = roles.into_map();
-    people_root_section.apply_roles(&role_map);
 
-    people_root_section.write(Path::new(&content_dir), Path::new(&content_sub_dir), 0)?;
+    let mut buckets = vec![
+        // Project leads
+        Vec::new(),
+        // Maintainers
+        Vec::new(),
+        // Other roles
+        Vec::new(),
+        // No roles
+        Vec::new(),
+    ];
+    let mut assets_to_copy = Vec::new();
+    for org_member in &org_members {
+        if let Some(ProfilePicture::File(file)) = org_member.profile_picture.as_ref() {
+            let filename = PathBuf::from(file).file_name().unwrap().to_os_string();
+            assets_to_copy.push(org_members_path.join(filename));
+        }
+    }
 
-    let Some(CommunityNode::Section(org)) = people_root_section.content.iter().find(|node| node.name() == "The Bevy Organization") else {
-        panic!("unexpected kind of node or missing for The Bevy Organization");
-    };
-    let mut donate = org.clone();
-    
-    donate.name = "Supporting Bevy Development".to_string();
-    donate.filename = Some("donate".to_string());
-    donate.header = Some("Supporting Bevy".to_string());
-    donate.template = Some("donate.html".to_string());
+    for member in &community_members {
+        if let Some(ProfilePicture::File(file)) = member.profile_picture.as_ref() {
+            let filename = PathBuf::from(file).file_name().unwrap().to_os_string();
+            assets_to_copy.push(community_members_path.join(filename));
+        }
+    }
 
-    donate.content = donate.content.into_iter().filter(|node| {
-        let CommunityNode::Member(member) = node else {
-            panic!("got an unexpected subsection");
-        };
-        member.sponsor.is_some()
-    }).collect();
+    for mut org_member in org_members {
+        let roles = org_member
+            .github
+            .as_ref()
+            .and_then(|github| role_map.get(github).cloned());
+        org_member.roles = roles.clone();
+        if let Some(r) = roles {
+            if r.iter().find(|p| *p == "Project Lead").is_some() {
+                buckets[0].push(org_member);
+            } else if r.iter().find(|p| *p == "Maintainer").is_some() {
+                buckets[1].push(org_member);
+            } else if r.is_empty() {
+                buckets[2].push(org_member);
+            } else {
+                buckets[3].push(org_member);
+            }
+        } else {
+            buckets[3].push(org_member);
+        }
+    }
+    let mut org_members = Vec::new();
+    for mut group in buckets {
+        group.shuffle(&mut rng);
+        org_members.extend(group);
+    }
 
-    donate.write(Path::new(&content_dir), Path::new(&content_sub_dir), 0)?;
+    // TODO: collect files to copy to static dir here
+
+    let org_members: Vec<_> = org_members
+        .into_iter()
+        .map(MemberOutput::from_member)
```

Comment:
```suggestion
        .into_iter()
        .filter(|member| member.sponsor.is_some())
        .map(MemberOutput::from_member)
```
The page this is used on needs only the members with a sponsor link